tests/server/signup.test.js

```diff
@@ -1,199 +1,156 @@
-import { insertUser } from "../../server/api/signup";
-import { db } from "../../server/db";
-import * as Crypto from "crypto";
-import Supertest from "supertest";
+const { db } = require("../../server/db");
+const crypto = require("crypto");
 
-const App = require("../../server/index");
-const request = Supertest(App);
+const app = require("../../server/index");
+const Supertest = require("supertest");
+const request = Supertest(app);
 
 jest.mock("../../server/db");
 jest.mock("../../server/api/login", () => {
   return {
-    login: jest.fn((req, res, next) => { return res.status(200).send("OK")})
+    login: jest.fn((req, response) => { return response.status(200).send("OK") })
```

Comment:
Change from `res` => `response` doesn't make sense in the face of `req` not being changed to `request`; seems like you just did a big search and replace.

Makes sense that `const response = await ...` but the nomenclature everywhere else for express callbacks (even in express documentation) is `(req, res, next) => {}`.

Reply:
Found again at lines 17 and 20.

Reply:
(also maybe in other files, just found it now)